apps/web/components/email-list/EmailList.tsx

```diff
@@ -472,6 +488,28 @@ export function EmailList(props: {
                   />
                 );
               })}
+              {/* Load more button to fetch more data */}
+              {showLoadMore && (
+                <div className="pb-2">
```

Comment:
styling for this button is off in your video. adding px-4 may fix this. you could also drop the wrapper div, and just do mb-2 mx-4 on the button

Reply:
Removed the wrapper div and added mb-2 for the button